packages/plugin-mashups/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@ciscospark/plugin-mashups",
+  "version": "0.7.0-alpha",
+  "description": "",
+  "main": "dist/index.js",
+  "devMain": "src/index.js",
+  "author": "arun <arungane@cisco.com>",
+  "license": "(Apache-2.0)",
+  "repository": "https://github.com/ciscospark/spark-js-sdk/tree/master/packages/plugin-mashups",
+  "dependencies": {
+    "@ciscospark/common": "^0.7.0-alpha",
+    "@ciscospark/spark-core": "^0.7.0-alpha",
+    "@ciscospark/plugin-wdm": "^0.7.0-alpha",
+    "@ciscospark/plugin-conversation": "^0.7.0-alpha",
```

Comment:
I'm pretty sure you only need plugin-conversation for your test setup; if so, you should move this to a dev dependency.